submissions/ustymchyk/memory-pair-game/index.js

```diff
@@ -0,0 +1,94 @@
+class Game {
+    constructor(selector) {
+        this.gameContainer = document.querySelector(selector);
+        this.rightAnswers = 0;
+        this.openCards = [];
+    }
+
+    startGame() {
+        this.gameContainer.innerHTML = this.getCards();
+        this.addListeners();
+    }
+
+    addListeners() {
+        this.gameContainer.querySelectorAll('[data-onclick]').forEach(button => {
+            const attr = button.getAttribute('data-onclick');
+            const methodName = attr.split('(')[0];
+
+            button.addEventListener('click', event => this[methodName](event));
+        });
+    }
+
+    openCard({ currentTarget: btn }) {
+        btn.classList.add('active');
+
+        if (this.openCards.length < 2) {
+            this.openCards.push(btn);
+        }
+        if (this.openCards.length === 2) {
+            this.checkMatch();
+        }
+    }
+
+    checkMatch() {
+        if (this.openCards[0].getAttribute('data-id') === this.openCards[1].getAttribute('data-id')) {
```

Comment:
you can use this.openCards[0].dataset.id here as well
Also, maybe using ternary operator will help to make this code shorter:
```suggestion
        this.openCards[0].dataset.id === this.openCards[1].dataset.id ?  this.addMatch() : this.closeCards()
```